pillar/params.sls

```diff
@@ -75,3 +75,7 @@ proxy:
 reboot:
   group:          'default'
   directory:      'opensuse.org/rebootmgr/locks'
+
+transactional-update:
+  timer:
+    OnCalendar: '*:0/15' # See https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events for syntax
```

Comment:
It is in the pillar - and yes - that is what it is expanded to. I can be more explicit if needed - but lets agree on a timescale.